.buildkite/pipelines/security_solution/security_solution_cypress.yml

```diff
@@ -1,108 +1,96 @@
 steps:
-  - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless
-    label: 'Serverless MKI QA Security Cypress Tests'
-    agents:
-      queue: n2-4-spot
-    # TODO : Revise the timeout when the pipeline will be officially integrated with the quality gate.
-    timeout_in_minutes: 300
-    parallelism: 6
-    retry:
-      automatic:
-        - exit_status: '*'
-          limit: 1
-
   - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless:explore
-    label: 'Serverless MKI QA Explore - Security Solution Cypress Tests'
+    label: "Serverless MKI QA Explore - Security Solution Cypress Tests"
     agents:
       queue: n2-4-spot
     # TODO : Revise the timeout when the pipeline will be officially integrated with the quality gate.
     timeout_in_minutes: 300
     parallelism: 4
     retry:
       automatic:
-        - exit_status: '*'
+        - exit_status: "*"
           limit: 1
 
   - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless:investigations
-    label: 'Serverless MKI QA Investigations - Security Solution Cypress Tests'
+    label: "Serverless MKI QA Investigations - Security Solution Cypress Tests"
     agents:
       queue: n2-4-spot
     # TODO : Revise the timeout when the pipeline will be officially integrated with the quality gate.
     timeout_in_minutes: 300
     parallelism: 8
     retry:
       automatic:
-        - exit_status: '*'
+        - exit_status: "*"
           limit: 1
 
   - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless:rule_management
-    label: 'Serverless MKI QA Rule Management - Security Solution Cypress Tests'
+    label: "Serverless MKI QA Rule Management - Security Solution Cypress Tests"
     agents:
       queue: n2-4-spot
     # TODO : Revise the timeout when the pipeline will be officially integrated with the quality gate.
     timeout_in_minutes: 300
     parallelism: 8
     retry:
       automatic:
-        - exit_status: '*'
+        - exit_status: "*"
           limit: 1
 
   - command: .buildkite/scripts/pipelines/security_solution_quality_gate/security_solution_cypress/mki_security_solution_cypress.sh cypress:run:qa:serverless:rule_management:prebuilt_rules
-    label: 'Serverless MKI QA Rule Management - Prebuilt Rules - Security Solution Cypress Tests'
+    label: "Serverless MKI QA Rule Management - Prebuilt Rules - Security Solution Cypress Tests"
     agents:
       queue: n2-4-spot
     # TODO : Revise the timeout when the pipeline will be officially integrated with the quality gate.
     timeout_in_minutes: 300
-    parallelism: 6
+    parallelism: 2
```

Comment:
We should update the parallelism also in:
* `.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml`
* `.buildkite/pipelines/on_merge.yml`
* `.buildkite/pipelines/pull_request/base.yml`

Reply:
Fixed in 7e1868b11fa1656c7a01aaab62a53fe6c374ad3c